runner/runner.go

```diff
@@ -1329,6 +1330,48 @@ func Grade(
 						},
 					)
 				}
+				// Check if we have an expected stderr message from the validator;
+				// fail the case with JE if there's a mismatch.
+				if runScore.Cmp(big.NewRat(1, 1)) != 0 {
+					expectedStderrPath := path.Join(
+						input.Path(), "cases", fmt.Sprintf("%s.expected-failure", caseData.Name),
+					)
+					expectedStderr, err := ioutil.ReadFile(expectedStderrPath)
+					if err != nil && !errors.Is(err, fs.ErrNotExist) {
+						ctx.Log.Warn(
+							"Error opening expected file",
+							map[string]interface{}{
+								"path": expectedStderrPath,
+								"err":  err,
+							},
+						)
+						continue
+					}
+					if err == nil {
+						validatorStderrPath := path.Join(runRoot, "validator", fmt.Sprintf("%s.err", caseData.Name))
+						validatorStderr, err := ioutil.ReadFile(validatorStderrPath)
+						if err != nil {
+							ctx.Log.Warn(
+								"Error opening expected file",
+								map[string]interface{}{
+									"path": expectedStderrPath,
+									"err":  err,
+								},
+							)
+							continue
+						}
+						if !strings.Contains(string(validatorStderr), string(expectedStderr)) {
+							ctx.Log.Warn(
+								"Validator stderr did not contain expected string",
+								map[string]interface{}{
+									"case name": caseData.Name,
+								},
+							)
+							runResult.Verdict = worseVerdict(runResult.Verdict, "JE")
```

Comment:
```suggestion
							runResult.Verdict = worseVerdict(runResult.Verdict, "VE")
```

esto no es error del juez, sino del validador.